### PR TITLE
[11.0] [mig] [WIP] base custom info 

### PR DESCRIPTION
**Pull request by fanha99**
WIP
checking odoo bug

### CODE REVIEW

File: base_custom_info/models/custom_info.py [hunk 1]
```diff
@@ -30,21 +29,6 @@ class CustomInfo(models.AbstractModel):
         auto_join=True, string='Custom Properties',
     )
 
-    # HACK: Until https://github.com/odoo/odoo/pull/10557 is merged
```
Comment (fanha99) [inline base_custom_info/models/custom_info.py]: with or without this block of code, still the same result.

File: base_custom_info/models/custom_info.py [hunk 2]
```diff
@@ -60,9 +44,6 @@ def _onchange_custom_info_template_id(self):
                 "res_id": self.id,
                 "value": prop.default_value,
             })
-            # HACK https://github.com/odoo/odoo/issues/13076
```
Comment (fanha99) [inline base_custom_info/models/custom_info.py]: is this issue fixed in v11?

**Comment by pedrobaeza** [comment]
I think this patch is related, but for v12: https://github.com/odoo/odoo/commit/fff578fa22530aa66081cc27e72f70f996fe9c6a

**Comment by pedrobaeza** [comment]
Why do you close it?

**Pull request closed without merging**